### PR TITLE
Fixes #36734 - Hide double title on Register Host page

### PR DESCRIPTION
**Pull request by kmalyjur**
image

Changed PageLayout so that breadcrumb, title, search bar and toolbar buttons work together even if some of the elements are missing.

### CODE REVIEW

File: webpack/assets/javascripts/react_app/routes/common/PageLayout/PageLayout.js [hunk 1]
```diff
@@ -25,58 +25,66 @@ const PageLayout = ({
...
-          </TextContent>
-        )}
-        {customBreadcrumbs ||
-          (breadcrumbOptions && <BreadcrumbBar {...breadcrumbOptions} />)}
-      </div>
-    </PageSection>
+}) => {
+  const title = (
+    <TextContent>
+      <Text ouiaId="title" component="h1">
```
Comment (MariaAga) [inline webpack/assets/javascripts/react_app/routes/common/PageLayout/PageLayout.js]: Can you keep it `breadcrumb_title`?

**Review by MariaAga — CHANGES_REQUESTED** [review]

**Comment by MariaAga** [comment]
@kmalyjur test failures look related

**Comment by kmalyjur** [comment]
@MariaAga I added an additional `PageSection` for the title mainly because of the case when breadcrumb and search bar are present, and it also fixes the previous failures elegantly.

**Comment by MariaAga** [comment]
can you please move the conditions to not show the page sections? so we wont end up with extra empty page sections that add space before the title.
Also, integration test failures are related.

**Review by MariaAga — APPROVED** [review]

**Comment by MariaAga** [comment]
Thanks!

**Pull request merged**